application/deed/model.py

```diff
@@ -22,3 +24,21 @@ def generate_token():
 
     def get_json_doc(self):
         return copy.deepcopy(self.json_doc)
+
+    def get_deed_status(title_number, mdref):
+
+        deeds = Deed.query.filter(
+            and_(
+                Deed.deed['title_number'].astext == title_number,
+                Deed.deed['md_ref'].astext == mdref
+            )
+        ).all()
```

Comment:
Completely agree, we must look at how to index json items within postgres as we scale as I doubt its possible in sqlalchemy. Good shout.